webplugin/template/chat.html

```diff
@@ -21,6 +21,7 @@
         }
         let params = new URLSearchParams(window.location.search);
         let appId = params.get('appId');
+        let isInputFocussed = params.get('focused')
```

Comment:
will this be passed by customer? please provide details on how is it going to be used

Reply:
yes, this is going to be passed by the customer, We will instruct the customer to add that param 

**For example** 

<iframe
            style="border: none;"
            height="600px"
            width="400px"
            src="https://widget.kommunicate.io/chat?appId=YOUR_APP_ID&focused=false"
            allow="microphone; geolocation;"
        >
</iframe>